scripts/create_custom_video_dataset.py

```diff
@@ -0,0 +1,105 @@
+import argparse
+import cv2
+import os
+import pandas as pd
+import shutil
+from PIL import Image
+
+def main(args):
```

Comment:
Two newline separation (PEP8)
```suggestion
from PIL import Image


def main(args):
```